test/fixtures/parser/bigquery/create_table_columns_partition_options.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE newtable (
+    x INT64,
+    y INT64
+)
+PARTITION BY x, y
+CLUSTER BY x, y
+OPTIONS(description="foo")
```

Comment:
Small nit: This is not a valid BigQuery query

```
Only a single PARTITION BY expression is supported but found 2
```

Maybe you could replace it with something valid? For example a something that is probably commonly used

```suggestion
CREATE TABLE newtable (
    x TIMESTAMP,
    y INT64
)
PARTITION BY DATE(x)
CLUSTER BY x, y
OPTIONS(description="foo")
```

Reply:
Good point -- I'll change this. Note that in general, SQLFluff dialects are often more permissive than the actual implementation, but that's usually for pragmatic reasons (speed or simplicity of implementation). The correct thing should be easy here, though. 👍🏽